integration-tests/bats/merge.bats

```diff
@@ -851,3 +846,23 @@ SQL
     log_status_eq 1
     [[ "$output" =~ "table with same name deleted and modified" ]] || false
 }
+
```

Comment:
line 817 has a good check with a merge that finds a constraint violation. Would be good to add a line there just to test that the commit didn't happen and that changes are in the working set.